### ISSUE TITLE
Quick Menu: Keyboard functionality

### ISSUE THREAD

**Issue by BrittanyIRL** [GoogleForCreators/web-stories-wp#7509]
Quick menu is a context menu that's always visible. 

This ticket is to make sure the keyboard functionality is 100% now that we can see it in the editor with its many focus traps. 

Right now, focus gets trapped in the menu, we should be able to continue tabbing through the page after the last menu item and also backwards through focus with tab + shift. 

(not really a bug since this feature is in flight, just a more specific ticket to isolate changes since it wasn't an issue in storybook) 

![trapped.gif](https://images.zenhubusercontent.com/5e9dcd9a9ff5dfdaffcdf132/564f8dbf-5de4-4990-90ca-05872b30ec8c)

**Comment by swissspidy** [comment]
> Right now, focus gets trapped in the menu, we should be able to continue tabbing through the page after the last menu item and also backwards through focus with tab + shift.

Navigation between menu items should probably happen with arrow keys instead (see how this works for library tabs and others). So you can easily tab in and out of the menu.

**Comment by BrittanyIRL** [comment]
Thanks, Pascal!

> Navigation between menu items should probably happen with arrow keys instead (see how this works for library tabs and others). So you can easily tab in and out of the menu. 

Menus are already navigable with the keyboard arrows, this is for getting _out_ of the menu with tab, so we should be good.

**Comment by maxyinger** [comment]
Just to clarify here, I think the expected behavior that's getting requested is:

- I tab into the quick menu and first item is focused (lets say the quick menu has 3 items)
- I press the down arrow, the second item in the menu now receives focus
- I press tab, the focus gets moved off the menu to the next keyboard navigable element (outside the menu)

This would require slight alterations to allow this to happen as right now, this is how the menu is built:
- second item is focused (lets say the quick menu has 3 items)
- either down arrow or tab would take you to the third item being focused
- pressing tab now that the third (and last item) is focused takes you out of the menu

Let me know if my understanding is correct or if the second explained flow is actually the expected behavior and needs no alterations?
